<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Log In</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>55</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>UserName: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>95</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Password: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnEnter">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>139</y>
     <width>260</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>ENTER</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cmbChoice">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>9</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtUsername">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>55</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="txtPassword">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>96</y>
     <width>260</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
